<?xml version="1.0" encoding="UTF-8"?>
<com.ibm.datatools.etl.errmsgs:Message xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:com.ibm.datatools.etl.errmsgs="http:///com/ibm/datatools/etl/errmsgs.ecore" msgNum="12207" timestamp="1516334753116" msgTxt="Validation failed for the 'l1q1p2'" component="dataflow.validation" msgDescTxt="The validation of this flow encountered an error." msgCause="" msgResponse="Assign a variable (if allowed) for this property or enter a fixed value." entityType="DataFlow">
  <expandedMsgs msgNum="12208" timestamp="1516334753116" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="03"/>
  <expandedMsgs msgNum="12208" timestamp="1516334753116" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="04">
    <expandedMsgs msgNum="1253" timestamp="1516334753116" msgTxt="Operator 'class': Error on condition 'SQL_?ColumnRefUnresolved : Unable to find table for column &quot;313&quot;.'" msgSeverity="WARN" component="dataflow.core" msgDescTxt="Attempting to parse condition with the SQL parser generated an error. " msgCause="A problem was encountered while parsing the condition. " msgResponse="Correct the condition." entityType="MSG_CONDITION_PARSE" operatorID="04"/>
  </expandedMsgs>
  <expandedMsgs msgNum="12208" timestamp="1516334753116" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="022"/>
  <expandedMsgs msgNum="12208" timestamp="1516334753116" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="023">
    <expandedMsgs msgNum="12202" timestamp="1516334753116" msgTxt="Operator 'Table Join_023': Input Port 'in1' is missing an incoming link." component="dataflow.codegen" msgDescTxt="Check input links " msgCause="The input port 'in1' is not connected." msgResponse="Connect the input port 'in1'. " entityType="CHECKINPUTLINK" operatorID="023"/>
  </expandedMsgs>
  <expandedMsgs msgNum="12208" timestamp="1516334753116" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="026"/>
  <expandedMsgs msgNum="12208" timestamp="1516334753116" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="030"/>
  <expandedMsgs msgNum="12208" timestamp="1516334753116" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="033"/>
</com.ibm.datatools.etl.errmsgs:Message>
